### PR TITLE
bioconda-utils: test requirements satisfaction

### PR DESCRIPTION
**Pull request by mbargull**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@dpryan79: Haven't tested this, but maybe this works as a regression test.
PS: This will create a merge conflict on `bulk`. I cherry-picked parts of your commit from `bulk`.

### CODE REVIEW

**Comment by dpryan79** [comment]
This is definitely worth a try

**Comment by mbargull** [comment]
The test failed! Which is good -- means it might work ;). Let's give the CI another go.

File: recipes/bioconda-utils/meta.yaml [hunk 1]
```diff
@@ -18,10 +18,10 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.7
```
Comment (dpryan79) [inline recipes/bioconda-utils/meta.yaml]: You can ignore this requirement in practice, I think it's a hold-over from some earlier testing.

File: recipes/bioconda-utils/meta.yaml [hunk 2]
```diff
@@ -78,7 +78,7 @@ test:
       --file="$( python -c
       'from os.path import dirname, join; import bioconda_utils; print(join(dirname(bioconda_utils.__file__), "bioconda_utils-requirements.txt"))'
       )"
-      --json | grep -qF 'All requested packages already installed.'
+      --json | tee /dev/stderr | grep -qF 'All requested packages already installed.'
```
Comment (mbargull) [inline recipes/bioconda-utils/meta.yaml]: IDK whether `tee /dev/stderr` works on macOS or not. We'll see what the CI says..

File: recipes/bioconda-utils/meta.yaml (hunk 1 above)
Comment (mbargull) [reply recipes/bioconda-utils/meta.yaml]: Possibly, but it doesn't hurt to go for 3.7.
For the test we definitely need to have the requirements in `meta.yaml` and `bioconda_utils-requirements.txt` in agreement, though.

**Pull request closed without merging**